starter/script.js

```diff
@@ -1 +1,33 @@
 // put code here!
+var tantrumFree = function() {
+  console.log("**************************")
+}
+console.log ("You've overslept one morning and realized you have 25 minutes to get yourself and your kindergartener out the door")
+var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button") (Please enter 'j' or 's')
```

Comment:
Similar to the comment I made a few lines below, this line has issues with quotations. The key thing to realize is that `prompt` is a function that takes one argument (a string). This means when we call it, we have to give it *all* the text we want it to show in 1 string:

```js
//incorrect:
// var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button") (Please enter 'j' or 's')

//correct:
   var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button (Please enter 'j' or 's')" )
```

Even though our string of text has parenthesis in them, it doesn't mean we need to end the string, they are *part of the string*, and distinct from the closing parenthesis that we use to *call* the function `prompt`.